iceberg-rust/src/view/transaction/operation.rs

```diff
@@ -47,9 +47,17 @@ impl Operation {
                 schema,
                 branch,
             } => {
+                let schema_changed = metadata.current_schema(branch.as_deref())
+                    .map(|s| schema != *s.fields())
```

Comment:
In theory `identifier_field_ids` can change while the fields remain the same, so we should also test for that when deducing a schema change?

Reply:
In theory you're right, but looks like iceberg-rs doesn't support changing identifier_field_ids, so identifier_field_ids are not even represented in variant Operation::UpdateRepresentation

Reply:
Oh I see; then no need to test it now.